### PR TITLE
[Metrics UI] Remove anomaly alert documentation

### PR DESCRIPTION
**Pull request by simianhacker**
This PR is part of https://github.com/elastic/kibana/issues/93645 and removes the mentions of anomaly alerts in the Metrics UI documentation. This also removed unused images and the anomaly alert specific documentation.

### CODE REVIEW

File: docs/en/observability/infrastructure-threshold-alert.asciidoc [hunk 1]
```diff
@@ -23,9 +23,8 @@ like to be notified about and then quickly create an alert in just a few clicks.
 == Inventory conditions
 
 Conditions for each alert can be applied to specific metrics relating to the inventory type you select.
-You can choose the aggregation type, the metric, and by including a warning threshold value, you can be
-alerted on multiple threshold values based on severity scores. When creating the alert, you can still get
-notified if no data is returned for the specific metric or if the alert fails to query {es}.
+You can choose the aggregation type, the metric, and by including a warning threshold value. When creating
+the alert, you can still get notified if no data is returned for the specific metric or if the alert fails to query {es}.
```
Comment (neptunian) [inline docs/en/observability/infrastructure-threshold-alert.asciidoc:27]: Why does this part need to change?  I'm not sure the first sentence makes sense any longer.

Comment (simianhacker) [reply docs/en/observability/infrastructure-threshold-alert.asciidoc]: Fixed

File: docs/en/observability/metrics-threshold-alert.asciidoc [hunk 2]
```diff
@@ -27,8 +27,8 @@ are automatically populated.
 == Metric conditions
 
 Conditions for each alert can be applied to specific metrics that you select. You can select the aggregation type,
-the metric, and by including a warning threshold value, you can be alerted on multiple threshold values based on severity scores.
-To help you determine which thresholds are meaningful to you, the preview charts provide a visualization.
+the metric, and by including a warning threshold value.  To help you determine which thresholds are meaningful to you,
```
Comment (neptunian) [inline docs/en/observability/metrics-threshold-alert.asciidoc:30]: same here as my comment above.

Comment (simianhacker) [reply docs/en/observability/metrics-threshold-alert.asciidoc]: fixed

**Comment by neptunian** [comment]
Can't approve it, but looks good to me.  Thanks, @simianhacker!

**Review by EamonnTP — APPROVED** [review]
@simianhacker Thanks for making these updates! The changes will need to be back ported to the `7.x` and `7.12` branches. Thanks again!

**Pull request merged**